Source/WebCore/platform/network/NetworkStorageSession.cpp

```diff
@@ -162,19 +162,25 @@ ThirdPartyCookieBlockingDecision NetworkStorageSession::thirdPartyCookieBlocking
     if (hasStorageAccess(resourceDomain, firstPartyDomain, frameID, pageID))
         return ThirdPartyCookieBlockingDecision::None;
 
+    const auto decideThirdPartyCookieBlocking = [isOptInCookiePartitioningEnabled = isOptInCookiePartitioningEnabled()] (bool shouldAllowUnpartitionedCookies) {
+        if (shouldAllowUnpartitionedCookies)
+            return ThirdPartyCookieBlockingDecision::None;
+        return isOptInCookiePartitioningEnabled ? ThirdPartyCookieBlockingDecision::AllExceptPartitioned : ThirdPartyCookieBlockingDecision::All;
```

Comment:
I think the enum value `AllExceptPartitioned` and `isOptInCookiePartitioningEnabled` flag might need to be protected by the compile time flag -- so it's only possible to set that value on available builds.